test/services/teams/events/index.test.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable new-cap */
+
+const { expect } = require('chai');
+const { ObjectId } = require('mongoose').Types;
+
+const app = require('../../../../src/app');
+const { userModel } = require('../../../../src/services/user/model');
+const { teamsModel, teamUserModel } = require('../../../../src/services/teams/model');
+
+const DELAY_TIME = 250;
+const schoolId = ObjectId();
+const userRoleId = ObjectId();
+const teamRoleId = ObjectId();
+
+const wait = () => new Promise((resolve) => {
```

Comment:
shorthand: `const wait = require('util').promisify(setTimeout);`

Reply:
Oh nice, thank you! :)